addons/web/static/src/views/kanban/kanban_cover_image_dialog.js

```diff
@@ -33,6 +33,10 @@ export class KanbanCoverImageDialog extends Component {
         });
     }
 
+    get hasCoverImage() {
+        return this.props.record.data[this.props.fieldName].length;
```

Comment:
remove `.length` (it's `false` when unset, so checking the length isn't necessary)

Reply:
indeed, thanks. 🙂